### PR TITLE
refactor: install steps

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -309,32 +309,28 @@
...
-          the manifest</a>.
-          </li>
-          <li>If <a>obtaining the manifest</a> results in an error, a user
-          agent can, at this point, fall back to using the <a>top-level
-          browsing context</a>' <code>Document</code>'s metadata to create an
-          <a>installation process</a>.
-          </li>
-          <li>If the <a>installation succeeded</a>, <a>fire an event</a> named
-          <code>install</code> at the <var>window</var> object.
+          <li>Then, <a>in parallel</a>:
```
Comment (kenchris) [inline index.html]: There is only one <ol> so why in parallel?

Comment (marcoscaceres) [inline index.html]: it's done off main thread.

Comment (kenchris) [inline index.html]: ah, in parallel with main thread.

lgtm then

Comment (marcoscaceres) [inline index.html]: 👍

**Pull request merged**